package.json

```diff
@@ -41,43 +41,43 @@
         "@hmcts/info-provider": "^1.1.0",
         "@hmcts/nodejs-healthcheck": "^1.8.3",
         "@hmcts/nodejs-logging": "^4.0.4",
-        "@hmcts/properties-volume": "^1.0.0",
+        "@hmcts/properties-volume": "^1.1.0",
         "@ministryofjustice/frontend": "^1.8.0",
-        "@types/config": "^3.3.1",
-        "@types/cookie-parser": "^1.4.4",
-        "@types/es6-promisify": "^6.0.2",
-        "@types/express": "^4.17.18",
+        "@types/config": "^3.3.3",
+        "@types/cookie-parser": "^1.4.6",
+        "@types/es6-promisify": "^6.0.4",
+        "@types/express": "^4.17.21",
         "@types/glob": "^8.1.0",
         "@types/helmet": "^4.0.0",
         "@types/i18next": "^13.0.0",
-        "@types/lodash": "^4.14.199",
-        "@types/node": "^20.8.2",
-        "@types/nunjucks": "^3.2.4",
-        "@types/require-directory": "^2.1.3",
-        "@types/serve-favicon": "^2.5.5",
-        "applicationinsights": "^2.8.0",
-        "awilix": "^8.0.1",
-        "axios": "^1.5.1",
+        "@types/lodash": "^4.14.201",
+        "@types/node": "^20.9.0",
+        "@types/nunjucks": "^3.2.6",
+        "@types/require-directory": "^2.1.5",
+        "@types/serve-favicon": "^2.5.7",
+        "applicationinsights": "^2.9.0",
+        "awilix": "^9.0.0",
+        "axios": "^1.6.1",
         "axios-oauth-client": "^1.5.0",
         "axios-token-interceptor": "^0.2.0",
-        "codeceptjs": "^3.5.5",
+        "codeceptjs": "3.5.6",
```

Comment:
Just checking the '^' symbol was removed intentionally to only accept the fixed version?

Reply:
Yeah that's correct. 3.5.7 was causing extreme slowness in the tests alongside timeouts. Can't see anything in the changelog that could have caused it, so potentially a bug.